features/dd-sdk-android-session-replay-compose/src/test/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/semantics/ComposeViewMapperTest.kt

```diff
@@ -84,41 +78,42 @@ class SemanticsWireframeMapperTest {
     @Forgery
     private lateinit var fakeMappingContext: MappingContext
 
-    private lateinit var testedSemanticsWireframeMapper: SemanticsWireframeMapper
+    private lateinit var testedComposeViewMapper: ComposeViewMapper
 
     @BeforeEach
     fun `set up`() {
-        testedSemanticsWireframeMapper = SemanticsWireframeMapper(
+        testedComposeViewMapper = ComposeViewMapper(
             mockViewIdentifierResolver,
             mockColorStringFormatter,
             mockViewBoundsResolver,
             mockDrawableToColorMapper,
             mockSemanticsUtils,
-            mapOf(),
-            mockTextSemanticsNodeMapper,
-            mockContainerSemanticsNodeMapper
+            mockRootSemanticsNodeMapper
         )
     }
 
     @Test
-    fun `M use TextSemanticsNodeMapper W map { role is missing }`() {
+    fun `M invoke rootSemanticsNodeMapper createComposeWireframes W map`() {
         // Given
         val mockSemanticsNode = mockSemanticsNode(null)
         whenever(mockSemanticsUtils.findRootSemanticsNode(mockView)).thenReturn(mockSemanticsNode)
 
         // When
-        testedSemanticsWireframeMapper.map(
+        testedComposeViewMapper.map(
             mockView,
             fakeMappingContext,
             mockAsyncJobStatusCallback,
             mockInternalLogger
         )
 
         // Then
-        verify(mockContainerSemanticsNodeMapper).map(
-            eq(mockSemanticsNode),
-            any(),
-            eq(mockAsyncJobStatusCallback)
+
+        mockRootSemanticsNodeMapper.createComposeWireframes(
```

Comment:
there is no verification/assertion here, is it expected?

Reply:
same as above